### PR TITLE
Autoselect uploaded asset in selector

### LINKED ISSUE #3602: Autoselect most recently uploaded asset
When uploading an asset to the container from the asset field "Browse" button I feel it should be automatically selected and the assets should have a default sort order of latest first rather than name.

That would mean you don't have to find the asset within the container after uploading.

### PR DESCRIPTION
**Pull request by arthurperton**
Fixes #3602.

* Automatically adds the uploaded asset to the selection. And sorts by last modified, most recent first.
* Enables sortable columns in the Asset Browser.

### CODE REVIEW

File: src/Http/Controllers/CP/Assets/BrowserController.php [hunk 1]
```diff
@@ -71,7 +71,9 @@ public function folder(Request $request, $container, $path = '/')
 
         $folder = $container->assetFolder($path);
 
-        $assets = $folder->queryAssets()->paginate(30);
+        $assets = $folder->queryAssets()
+            ->orderBy($request->sort ?? 'basename', $request->order ?? 'asc')
```
Comment (jasonvarga) [inline src/Http/Controllers/CP/Assets/BrowserController.php:75]: Can you wrap `orderBy` in an `if` for whether `sort` is in the request?

It'll probably save overhead by not ordering if it doesn't need to.

**Review by jasonvarga — CHANGES_REQUESTED** [review]

File: src/Http/Controllers/CP/Assets/BrowserController.php (hunk 1 above)
Comment (arthurperton) [reply src/Http/Controllers/CP/Assets/BrowserController.php]: Sure!

**Review by jackmcdade — APPROVED** [review]
👌

**Comment by jackmcdade** [comment]
The masses will rejoice on this one! Thanks!

**Pull request merged**